### DISCUSSION TITLE
Flaky tests?

### DISCUSSION THREAD

**Discussion by rosano** [capricorn86/happy-dom#1091] [Q&A] [answered]
Hi, I found Happy DOM from bun.sh. Looks interesting but reminds me of [@azzaf/zombie](https://github.com/assaf/zombie) and as I used that for a while I wonder if flaky tests would also be an issue for this library? No idea what the technical differences are so forgive me if the question doesn't make sense. Just curious if there are known limitations or considerations for UI testing without a kind of headless browser.

**Comment by capricorn86** [comment, marked as answer]
Hi @rosano! :slightly_smiling_face: 

I believe that the main difference between Zombie and Happy DOM is that Happy DOM has implemented a lot more functionality from the WHATWG DOM and HTML spec. Another key difference is that Happy DOM is still maintained. However, all functionality from DOM and HTML hasn't been implemented yet and in those cases you may get some "flakiness".

You are very welcome to report (or even better if you contribute with a fix) if you find missing functionality or a bug.
